README.md

```diff
@@ -2,9 +2,17 @@
 
 [Mattermost](https://mattermost.com) is an open source platform for secure collaboration across the entire software development lifecycle. This repo is the primary source for core development on the Mattermost platform; it's written in Go and React and runs as a single Linux binary with MySQL or PostgreSQL. A new compiled version is released under an MIT license every month on the 16th.
 
+[Use it for free in Mattermost Cloud](https://mattermost.com/sign-up/) or [deploy on-premises](https://mattermost.com/deploy/).
```

Comment:
Should we add utm_source tags to these too?

Reply:
+1, good call. Added.